lib/requirefy.js

```diff
@@ -17,17 +17,29 @@ module.exports = function requirefy(code, options, resolvePath) {
     return header;
   }
 
-  function removeRequirejs(code, head, tail) {
+  function removeRequirejs(code, head, tail, moveUseStrict, useStricRange) {
     var top = code.substring(0, head.start - 1);
     var mid = code.substring(head.end, tail.start);
     var bot = code.substring(tail.end);
 
+    if (moveUseStrict) {
+      var beforeUseStrict = mid.substring(0, useStricRange[0]);
```

Comment:
I guess you could, the difference is that `slice` makes you think about a string the same way as about an array.
The `substring` method is totally superfluous, but they do the same thing.

I've been using `slice` everywhere, but no big deal.